src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksDtoToPaymentWorksVendorConversionServiceImpl.java

```diff
@@ -94,17 +94,52 @@ private void populateNewVendorCorporateAddressAttributes(PaymentWorksVendor stgN
     }
 
     private void populateNewVendorRemittanceAddressAttributes(PaymentWorksVendor stgNewVendor, PaymentWorksRequestingCompanyDTO pmwRequestingCompanyDTO) {
+        PaymentWorksRemittanceAddressDTO pmwRemittanceAddressDTO = null;
         if (singleRemittanceAddressExists(pmwRequestingCompanyDTO)) {
-            PaymentWorksRemittanceAddressDTO pmwRemittanceAddressDTO = pmwRequestingCompanyDTO.getRemittance_addresses().getRemittance_address().get(0);
-            stgNewVendor.setRemittanceAddressStreet1(pmwRemittanceAddressDTO.getStreet1());
-            stgNewVendor.setRemittanceAddressStreet2(pmwRemittanceAddressDTO.getStreet2());
-            stgNewVendor.setRemittanceAddressCity(pmwRemittanceAddressDTO.getCity());
-            stgNewVendor.setRemittanceAddressState(pmwRemittanceAddressDTO.getState());
-            stgNewVendor.setRemittanceAddressCountry(pmwRemittanceAddressDTO.getCountry());
-            stgNewVendor.setRemittanceAddressZipCode(pmwRemittanceAddressDTO.getZipcode());
-            stgNewVendor.setRemittanceAddressValidated(pmwRemittanceAddressDTO.getValidated());
+            LOG.info("populateNewVendorRemittanceAddressAttributes: Single remit address found.");
+            pmwRemittanceAddressDTO = pmwRequestingCompanyDTO.getRemittance_addresses().getRemittance_address().get(0);
+            populateEachNewVendorRemitAddressAttribute(stgNewVendor, pmwRemittanceAddressDTO);
             populateNewVendorBankAccountAttributes(stgNewVendor, pmwRemittanceAddressDTO);
         }
+        else {
+            pmwRemittanceAddressDTO = determineRemitAddressToUse(pmwRequestingCompanyDTO);
+            if (ObjectUtils.isNotNull(pmwRemittanceAddressDTO)) {
+                LOG.info("populateNewVendorRemittanceAddressAttributes: Remit addresses found to use from MULTIPLE remit addresses.");
+                populateEachNewVendorRemitAddressAttribute(stgNewVendor, pmwRemittanceAddressDTO);
+                populateNewVendorBankAccountAttributes(stgNewVendor, pmwRemittanceAddressDTO);
+            }
+            else {
```

Comment:
Another "else" that should be on the same line as the closing curly brace above it.